ods_ci/tests/Resources/CLI/ModelServing/llm.resource

```diff
@@ -63,7 +61,7 @@ Set Project And Runtime
     Deploy Serving Runtime    namespace=${namespace}    runtime=${runtime}    protocol=${protocol}
     IF   ${enable_metrics} == ${TRUE}
         Oc Apply    kind=ConfigMap    src=${UWM_CONFIG_FILEPATH}
-        Oc Apply    kind=ConfigMap    src=${UWM_ENABLE_FILEPATH}
+        Oc Apply    kind=ConfigMap    src=${MONITORING_CONFIG_FILEPATH}
```

Comment:
Beware there are now two keywords that can be handy in Common.robot: `Enable User Workload Monitoring` and `Configure User Workload Monitoring`

Reply:
Yes, better using them. Will do the change, thanks

Reply:
Done!